cmd/podman/images/sign.go

```diff
@@ -48,6 +48,10 @@ func init() {
 	flags.StringVar(&signOptions.CertDir, certDirFlagName, "", "`Pathname` of a directory containing TLS certificates and keys")
 	_ = signCommand.RegisterFlagCompletionFunc(certDirFlagName, completion.AutocompleteDefault)
 	flags.BoolVarP(&signOptions.All, "all", "a", false, "Sign all the manifests of the multi-architecture image")
+
+	authFileFlagName := "auth-file"
```

Comment:
Please remove the dash here and anywhere that I might have missed.
```suggestion
	authFileFlagName := "authfile"
```